iceoryx_hoofs/include/iceoryx_hoofs/cxx/stack.hpp

```diff
@@ -23,7 +23,7 @@ namespace iox
 {
 namespace cxx
 {
-// AXIVION Construct Ruleset-A12.1.1 : it is guaranteed that the array elements are initialized on access
+// AXIVION Construct AutosarC++19_03-A12.1.1 : it is guaranteed that the array elements are initialized on access
```

Comment:
@elBoberido I don't get `A12.1.1` reported in `stack` when removing this rule and running the check locally.

@dkroenke neillangmead41 Could you check that?

Reply:
Just to be on the same page which warning is shown on the dashboard
![image](https://user-images.githubusercontent.com/56729607/177787913-c6756836-1b5c-489a-aacc-9d098b83c680.png)

cc @mossmaurice @saif-at-github @dkroenke